lib/workflows/create_project.py

```diff
@@ -0,0 +1,53 @@
+from typing import List, Optional
+from openstack.connection import Connection
+
+from workflows.create_external_project import create_external_project
+from workflows.create_internal_project import create_internal_project
+
+
+def create_project(
+    conn: Connection,
+    project_name: str,
+    project_email: str,
+    project_description: str,
+    project_domain: str,
+    networking_type: str,
+    number_of_floating_ips: int = 1,
+    number_of_security_group_rules: int = 200,
+    project_immutable: Optional[bool] = False,
+    parent_id: Optional[str] = None,
+    admin_user_list: Optional[List[str]] = None,
+    user_list: Optional[List[str]] = None,
+):
+    if project_domain != "default":
+        raise RuntimeError(
+            "domain support is work in progress, please use default domain",
+        )
+    return (
```

Comment:
Whilst this is brief I feel like it's a bit clunky readability-wise. Could we do something like:


```
if networking_type == "internal":
           create_internal_project(...)
else if networking_type == "external" or networking_type == "jasmin"
           create_external_project(...)
else:
           raise NotImplementedError(f"Unknown networking type {networking_type}")
```